manifests/m/Martchus/syncthingtray/1.4.1/Martchus.syncthingtray.installer.yaml

```diff
@@ -1,16 +1,15 @@
-# Created with Komac v1.6.0
 # yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
 
 PackageIdentifier: Martchus.syncthingtray
 PackageVersion: 1.4.1
 InstallerType: zip
 NestedInstallerType: portable
 NestedInstallerFiles:
-- RelativeFilePath: syncthingctl-qt5-1.4.1-x86_64-w64-mingw32.exe
+- RelativeFilePath: syncthingtray-qt5-1.4.1-x86_64-w64-mingw32.exe
 ReleaseDate: 2023-04-06
 Installers:

```

Comment:
It would supposedly also make sense to add entries for the i686 build (if that's easily possible considering that the relative file path would then change as well).